internal/controller/component_build_controller.go

```diff
@@ -201,6 +203,21 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	if getContainerImageRepositoryForComponent(&component) == "" {
 		// Container image must be set. It's not possible to proceed without it.
 		log.Info("Waiting for ContainerImage to be set")
+
+		buildStatus := readBuildStatus(&component)
+
+		if buildStatus.Message == waitForContainerImageMessage {
+			return ctrl.Result{}, nil
+		}
+
+		buildStatus.Message = waitForContainerImageMessage
+		writeBuildStatus(&component, buildStatus)
+		if err := r.Client.Update(ctx, &component); err != nil {
+			log.Error(err, "failed to update component after waiting for containerImage", l.Action, l.ActionUpdate, l.Audit, "true")
```

Comment:
```suggestion
			log.Error(err, "failed to update component status message for waiting for containerImage", l.Action, l.ActionUpdate, l.Audit, "true")
```

Reply:
I'll keep mine as it is consistent with other update messages

I want it to be specific that it is 'after' something